media-libs/libjxl/libjxl-9999.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit xdg cmake git-r3
+
+DESCRIPTION="JPEG XL image format reference implementation"
+HOMEPAGE="https://github.com/libjxl/libjxl"
+
+EGIT_REPO_URI="https://github.com/libjxl/libjxl.git"
+
+LICENSE="BSD"
+SLOT="0"
+IUSE="examples"
+
+DEPEND="app-arch/brotli
+	sys-libs/zlib
+	media-libs/libpng
+	virtual/jpeg
+	virtual/opengl
+	media-libs/freeglut
+	media-libs/giflib
+	media-libs/openexr:=
+	dev-util/google-perftools
+	x11-misc/shared-mime-info
+	dev-cpp/gtest
+	dev-cpp/highway
+	media-libs/lcms
+"
+
+BDEPEND=""
+
+RDEPEND="${DEPEND}"
+
+src_configure() {
+	local mycmakeargs=(
+		-DCMAKE_SKIP_RPATH=ON
+		-DBUILD_TESTING=OFF
+		-DBUILD_SHARED_LIBS=ON
```

Comment:
Eclass passes this for us too, right?

Reply:
So you suggest to remove the `-DBUILD_SHARED_LIBS=ON` line?